packages/models-library/src/models_library/project_networks.py

```diff
@@ -0,0 +1,59 @@
+from models_library.projects import ProjectID
+from pydantic import BaseModel, Field, constr, validate_arguments
+
+from .generics import DictModel
+from .projects_nodes_io import NodeIDStr
+
+SERVICE_NETWORK_RE = r"^[a-zA-Z]([a-zA-Z0-9_-]{0,63})$"
+
+PROJECT_NETWORK_PREFIX = "prj-ntwrk"
+
+DockerNetworkName = constr(regex=SERVICE_NETWORK_RE)
+DockerNetworkAlias = constr(regex=SERVICE_NETWORK_RE)
+
+
+@validate_arguments
+def validate_network_name(value: DockerNetworkName) -> DockerNetworkName:
+    return value
+
+
+@validate_arguments
+def validate_network_alias(value: DockerNetworkAlias) -> DockerNetworkAlias:
+    return value
+
+
+class ContainerAliases(DictModel[NodeIDStr, DockerNetworkAlias]):
```

Comment:
the word alias suggests me that I can have multiple of them ... which is not the case here. Would not be more appropriate something like "*_display_name"  or "*_user_defined_name" ... ??